src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -175,6 +184,11 @@ private String unescapeChars(String escapedString) {
     return unescapedString;
   }
 
+  private boolean isWildcardPattern(String inputString) {
```

Comment:
This is only used by schema name right? Maybe better name it as isSchemaNameWildcardPattern